Detectors/MUON/MCH/Base/include/MCHBase/Digit.h

```diff
@@ -47,13 +47,16 @@ class Digit
 
   Digit() = default;
 
-  Digit(int detid, int pad, unsigned long adc, Time time);
+  Digit(int detid, int pad, unsigned long adc, Time time, uint16_t nSamples = 1);
   ~Digit() = default;
 
   bool operator==(const Digit&) const;
 
   Time getTime() const { return mTime; }
 
+  uint16_t getNSamples() const { return mNSamples; }
```

Comment:
(this can wait for a next PR) 
- for consistency with SampaCluster itself I would use nofSamples() 
- is the setNSamples method needed ? I don't see it used so far.

Reply:
Ok for renaming the function to `nofSamples()`.

The setter method will be needed once we commit the digits merger code, because the merged digits will have a number of samples equal to the sum of all original digits.